src/api.js

```diff
@@ -287,8 +287,8 @@ builder.declare({
   }
 
   // Not all webhook payloads include an e-mail for the user who triggered an event
-  let headUser = msg.details['event.head.user.id'].toString();
-  let userDetails = (await instGithub.users.getById({id: headUser})).data;
```

Comment:
Is the `id` field no longer available?  For users `id` is permanent, but `username` can change, so using the `id` is preferred where possible.

Reply:
Apparently `getById` is undocumented? In [the release notes](https://github.com/octokit/rest.js/releases/tag/v16.0.0) they have a way to replace it with a generic request but they seem to think that the endpoint could go away at any time.

Reply:
Hm, I guess GitHub isn't worried about this issue, then..